quaternionmedia/codecartographer
# 21 integrate with graphbase project

When a graph is plotted it is now saved in the database. 
Added "Saved Graphs" to home page to see the graphs in the database (currently any of the graphs previously plotted)

## Test:
### Check new Saved Games link
1. get latest code
2. docker compose up
3. go to site (2000)
4. click "Saved Graphs"
    - Will see message saying "No graphs in database"
5. click "Return to Home"

### Save a Graph to database
1. click "Parser"
2. enter github repo url
3. expand "contents"
4. find .py file
5. click "Plot"
6. click "Single Layout" or "Grid Layout"
7. click "Return to Home"
8. click "Saved Graphs"
    - will see the name of the file plotted
    - will see link "PLOT" next to file name

### Plot a graph from database
1. click "PLOT" next to the graph name (filename) in "Saved Graphs"
2. click "single layout" or "grid layout"
    - will see the same graph as before with edge arrows 
    
Closes #21
